### PR TITLE
Add support for negative values in RangeInt token

### PR DESCRIPTION
**Pull request by yblein**

### CODE REVIEW

**Comment by yblein** [comment]
It looks like Travis is failing on the leak tests. This is no suprise since these tests are randomly failing on my computer.

**Comment by yblein** [comment]
Il looks like the leak tests are failing when GOMAXPROCS is greater than 1, which is the case since Go 1.5 (GOMAXPROCS default is now the number of CPUs).

**Comment by zimmski** [comment]
I will take a look

File: fuzz/filter/positiveboundaryvalueanalysis.go [hunk 1]
```diff
@@ -71,13 +70,27 @@ func (f *PositiveBoundaryValueAnalysisFilter) Apply(tok token.Token) (token.Toke
 
 		// middle
 		if l > 2 {
-			if err := tok.Permutation(uint(math.Ceil(float64(l) / 2.0))); err != nil {
-				panic(err)
+			if tok.From() < 0 && tok.To() > 0 {
+				// the boundaries are be -1, 0 and 1
```
Comment (zimmski) [inline fuzz/filter/positiveboundaryvalueanalysis.go]: Since this changes the behaviour of the filter, please document the change above.

Comment (zimmski) [inline fuzz/filter/positiveboundaryvalueanalysis.go]: Also, remove the "be" from the comment.

File: parser/tavor.go [hunk 2]
```diff
@@ -1525,11 +1525,26 @@ func (p *tavorParser) parseTypedTokenDefinition(variableScope *token.VariableSco
...
+			} else if c == '+' {
+				c = p.scan.Scan()
+			}
+
 			log.Debugf("parseTypedTokenDefinition argument value %d:%v -> %v", p.scan.Line, scanner.TokenString(c), p.scan.TokenText())
 
 			switch c {
 			case scanner.Ident, scanner.String, scanner.Int:
-				arguments[arg] = p.scan.TokenText()
+				prefix := ""
```
Comment (zimmski) [inline parser/tavor.go]: Remove the prefix from here

File: parser/tavor.go [hunk 3]
```diff
@@ -1525,11 +1525,26 @@ func (p *tavorParser) parseTypedTokenDefinition(variableScope *token.VariableSco
 			}
 
 			c = p.scan.Scan()
+
+			// optional sign (+/-)
+			neg := false
```
Comment (zimmski) [inline parser/tavor.go]: and add the prefix here. There is no need for the "neg" variable. Just the "prefix" variable is fine.

**Pull request merged**